### PR TITLE
Refactor Member Ignoring Method smell in HelpDialog class

### PR DESCRIPTION
**Pull request by emaiannone**
Hi, I'm Emanuele Iannone, a master student at University of Salerno.
Since my bachelor's thesis I have been working on a **code smell refactoring plugin** called *aDoctor*, which is able to identify and fix energy-related problems in Android apps.
I launched it on your project, finding different instances of code smells. I chose one of them and let the plugin automatically fix it.
In this case I chose Member Ignoring Method, that is present when a non static method does not use at all instance variables and other non static methods. These kind of smell may have a non trivial impact on energy consumption, as shown in this paper: `https://www.sciencedirect.com/science/article/pii/S0950584918301678`.
Besides, this kind of refactoring does not impact on the functionalities of your app, so it is totally safe. Let me know if you are interested in this refactoring proposal.

### CODE REVIEW

**Comment by forrestguice** [comment]
Hi. Thanks for your interest. If I understand correctly, this method can be static (and so it should be) as the compiled bytecode is demonstratably faster).

That's fine, but I'm not convinced these "smells" have "non trivial impact on energy consumption" and would be tempted to classify fixing them as premature optimization. If I made a point of eliminating every smell it would certainly have a non trivial impact on MY own energy consumption. I will need to see a graph containing bottlenecks at these lines before I allow these inefficiencies to bother me.

That said, I recognize the usefulness of flagging these issues, and am intrigued by your utility. I may run aDoctor on my own (the same way I periodically inspect the lint report).

File: app/src/main/java/com/forrestguice/suntimeswidget/HelpDialog.java [hunk 1]
```diff
@@ -96,20 +96,20 @@ public void onResume()
         expandSheet(getDialog());
     }
 
-    private void expandSheet(DialogInterface dialog)
-    {
```
Comment (forrestguice) [inline app/src/main/java/com/forrestguice/suntimeswidget/HelpDialog.java]: Strive to make your changes as succinct as possible. The only meaningful change here is the addition of the `static` keyword. Avoid whitespace only changes (style) as this just makes the diff harder to read.

File: app/app.iml [hunk 2]
```diff
@@ -4,6 +4,8 @@
     <facet type="android-gradle" name="Android-Gradle">
       <configuration>
         <option name="GRADLE_PROJECT_PATH" value=":app" />
+        <option name="LAST_SUCCESSFUL_SYNC_AGP_VERSION" />
```
Comment (forrestguice) [inline app/app.iml]: Don't commit changes to the AndroidStudio project files (as they are modified every time its run). They are only included in the repo to assist opening the project for the first time (and should arguably should be removed as they are completely optional to the build process and just making a mess).

**Comment by emaiannone** [comment]
Thank you for your feedback. I chose a single smell instance of the simplest kind (MIM). Of course the single smell isn't a real deal, but I didn't want to show your the complete fix because it might be harder to evaluate.
Regarding the changes in other files that was my fault when I did the commit, sorry :(

**Pull request closed without merging**